corehq/apps/app_manager/templates/app_manager/partials/supported-languages.html

```diff
@@ -52,10 +52,12 @@
         </tr>
     </tbody>
 </table>
-<button class="btn" data-bind="click: addLanguage, visible: editing">
-    <i class="icon-plus"></i>
-    Add Language
-</button>
+<div class="row-fluid">
```

Comment:
super minor -- the rest of the file uses `row`. does `row` in this case break the layout? if not i'd rather use that since `row-fluid` is not compatible in bootstrap 3 and the less files we have to touch the better